src/utils/number.ts

```diff
@@ -0,0 +1,3 @@
+export function isValidNumber(numberString: string): boolean {
+  return !numberString || isNaN(Number(numberString)) ? false : true;
```

Comment:
Would this work?

`!numberString || !isNaN(Number(numberString));`

Reply:
That won't work because the function needs to return `false` when the result of calling `isNaN` is true, so its the exact reverse.